modules/engine/test/shader-inputs.spec.ts

```diff
@@ -62,3 +63,35 @@ test('ShaderInputs#picking prop merge', t => {
 
   t.end();
 });
+
+[true, false].map(callback =>
```

Comment:
Nits:
- I find it disorienting to run `test` inside loops like this. I know there are still some instances of it left, but I prefer to have `test` on the top level and the loop inside.